libs/core/src/lib/multi-input/multi-input.component.ts

```diff
@@ -130,6 +143,9 @@ export class MultiInputComponent
     @Input()
     filterFn = this._defaultFilter;
 
+    @Input()
```

Comment:
could you please add comment description

Reply:
done